Package.swift

```diff
@@ -7,7 +7,7 @@ let package = Package(
     name: "ArcUI",
     platforms: [
         // .macOS(.v12), Uses UIKit
-        .iOS(.v15)
+        .iOS(.v16)
```

Comment:
If it's going into `develop` we need to support apps that haven't had an OS update so keeping this `v15`.
If it was a `release/v1.0.0` though it would be fine. Either is fine. For now easiest might be to keep this `v15`.

Reason CubeFoundation was `v16` was because

- We made a release branch for it for apps to migrate to
- It has some iOS 15 stuff that needs to be removed

Reply:
Okay, I've reverted the iOS version back to v15.0 and updated the dependency `CubeFoundation` branch to `release/v1.0.0`

Reply:
Reverted back to iOS v15